apps/site/snippets/en/download/nvm.bash

```diff
@@ -1,6 +1,9 @@
 # Download and install nvm:
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
 
+# in lieu of restarting the shell
+\. "$HOME/.nvm/nvm.sh"
```

Comment:
Just OOC, this will not break this time, right? Hahaha

Reply:
cc @ljharb

Reply:
correct - it was the `${  }` syntax (which is valid shell syntax) causing the trouble.

Long term, these files should probably not use JS template syntax since it overlaps with valid shell commands :-)

Reply:
> Long term, these files should probably not use JS template syntax since it overlaps with valid shell commands :-)

They can be escaped. Since it is a template syntax we can escape the quotes and special symbols 🙈

Reply:
The escaping slash, however, would appear in the output - so the escaping doesn't work properly.